docs/root/start/sandboxes/route-mirror.rst

```diff
@@ -0,0 +1,146 @@
+.. _install_sandboxes_route_mirror:
+
+Route mirroring policies
+========================
+
+.. sidebar:: Requirements
+
+   .. include:: _include/docker-env-setup-link.rst
+
+This simple example demonstrates Envoy's request mirroring capability using
+`request mirror policies <https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto#envoy-v3-api-msg-config-route-v3-routeaction-requestmirrorpolicy>`__.
+
+Incoming requests are received by ``envoy-front-proxy`` service. The envoy instance running
+in this container is configured with two routes:
+
+.. literalinclude:: _include/route-mirror/front-envoy.yaml
+   :language: yaml
+   :lines: 7-37
```

Comment:
lets reduce the displayed lines - i would probably start on the `virtual_hosts` line - but i think at most 2-3 lines before and 1 after the highlighed text

lets also add a caption with download (you should be able to find existing examples)